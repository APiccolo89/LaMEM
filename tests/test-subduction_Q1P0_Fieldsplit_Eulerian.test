EXEC            = ../bin/LaMEM -ParamFile input/subduction3D.dat  -restart 0 -vpt_element Q1P0 -time_end 1 -GridAdvectionMethod 0 -AddRandomNoiseParticles 0 -vs_fieldsplit_mg_levels_pc_type sor -vs_fieldsplit_1_pc_type sor
NPROCS          = 4
TEST_TYPE       = tolerance_diff_pl,1.0e-4

