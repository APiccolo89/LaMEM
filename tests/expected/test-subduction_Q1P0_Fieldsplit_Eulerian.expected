# -------------------------------------------------------------------------- 
#                   Lithosphere and Mantle Evolution Model                   
#              Current Revision: 4812 - 2013-10-01 23:57:18		     
#  Modified items: LaMEM_Particles.c, Elements.c, Assembly_FDSTAG.c, fdstag/JacRes.c, 
#     Compiled: Date: Oct  9 2013 - Time: 23:02:54 - Mode:  	    
# -------------------------------------------------------------------------- 
 Parsing input file : input/subduction3D.dat 
# Adding PetscOption: -vpt_element FDSTAG 
# Adding PetscOption: -MaximumSurfaceAngle 45 
# Adding PetscOption: -save_breakpoints 0 
# Adding PetscOption: -incr_breakpoints 100 
# Adding PetscOption: -AddRandomNoiseParticles 0 
# Adding PetscOption: -Air_ThicknessFactor 0.05 
# Adding PetscOption: -Slab_ThicknessFactor 0.1 
# Adding PetscOption: -use_stokes_norm L2 
# LithColDepth[0] = 0 
# LithColDens[0] = 0 
# LithColDens[1] = 0 
# --------------------------------------------------------------------------
# Phase material parameters read from input/subduction3D.dat: 
#    phase_0 (id=0), VISCOSITY, constant [0,0,0] = 1.000000e+20 
#    phase_0 (id=0), ELASTICITY, constant [0,1,0] = 5.000000e+10, 1.000000e+00 
#    phase_0 (id=0), DENSITY, temperature-dependent [0,2,0] = 3.100000e+03, 0.000000e+00, 0.000000e+00 
#    phase_0 (id=0), ENERGY, constant [0,3,0] = 3.000000e+00 1.050000e+03 0.000000e+00 0.000000e+00 
#    phase_1 (id=1), VISCOSITY, constant [1,0,0] = 1.000000e+22 
#    phase_1 (id=1), ELASTICITY, constant [1,1,0] = 5.000000e+10, 1.000000e+00 
#    phase_1 (id=1), DENSITY, temperature-dependent [1,2,0] = 3.200000e+03, 0.000000e+00, 0.000000e+00 
#    phase_1 (id=1), ENERGY, constant [1,3,0] = 3.000000e+00 1.050000e+03 0.000000e+00 0.000000e+00 
#    phase_2 (id=2), VISCOSITY, constant [2,0,0] = 1.000000e+20 
#    phase_2 (id=2), ELASTICITY, constant [2,1,0] = 5.000000e+10, 1.000000e+00 
#    phase_2 (id=2), DENSITY, temperature-dependent [2,2,0] = 3.100000e+03, 0.000000e+00, 0.000000e+00 
#    phase_2 (id=2), ENERGY, constant [2,3,0] = 3.000000e+00 1.050000e+03 0.000000e+00 0.000000e+00 
#    phase_3 (id=3), VISCOSITY, constant [3,0,0] = 1.000000e+20 
#    phase_3 (id=3), ELASTICITY, constant [3,1,0] = 5.000000e+10, 1.000000e+00 
#    phase_3 (id=3), DENSITY, temperature-dependent [3,2,0] = 3.100000e+03, 0.000000e+00, 0.000000e+00 
#    phase_3 (id=3), ENERGY, constant [3,3,0] = 3.000000e+00 1.050000e+03 0.000000e+00 0.000000e+00 
 Stokes solver             : Fully Coupled
 Velocity solver           : Fieldsplit + Algebraic Multigrid (ML)
 Element-type used         : Q1P0 ;  [Q1P0 Q2Pm1_global Q2Pm1_local Q1Q1 FDSTAG] (change with -vpt_element)
 Total # of cpu's          : 4 
 Total # of velocity dof's : 6534 
 Computing mean material per element using arithmetic averaging from integration points 
 Employing code in an Eulerian manner 
 Gravity angle with z-axis =  0 
 Approximate Aspect ratio of cells [dx/dz, dy/dz] = [4 1.00114] 
 Number of tracers/cell in [x,y,z]-direction = [3,3,3] 
 NumSurfaceNodes=9  nz=33 
 # Not employing nonlinear iterations 
 BC employed: BC.[Left Right; Front Back; Lower Upper]=[1 1; 1 1; 1 1] 
# Initial time step: 0.001 years 
 Input is assumed to be in dimensional units: 
  Characteristic Length 	= 1e+06 [m] 
  Characteristic Stress 	= 1e+06 [Pa] 
  Characteristic Temperature 	= 1000  [Celcius/K] 
  Characteristic Viscosity 	= 1e+20 [Pa.s] 
  Characteristic Time 	= 1e+14 [s] 
 Fine Grid parameters, nel=[32,1,32], cpu=[2,1,2] 
# Finished initializing Stokes matrices : 0.01143 s 
# Not adding random noise to initial particle distribution. 
Initialized particles 
#rank=0, Attributing 6912 particles to elements took 0.00775599 s 
The total # of particles present is 27648 
SlabThickness [-Slab_ThicknessFactor]: 0.070000 AirThickness [-Air_ThicknessFactor]: 0.035000 H: 0.700000 SlabWidthFactor [-Slab_WidthFactor]: 1.000000 
Saved initial particle distribution to directory ./InitialParticles 
The total memory used by integration points is 2.162688e+00 MB
The max memory used by integration points is 5.406720e-01 MB
The min memory used by integration points is 5.406720e-01 MB
#  Interpolating meshes and temperatures took 0.0014658 s 
#  Forming FEM stiffness matrix mx,my,mz=(33, 2, 33) ...   finished. [0.0141649 s]
 minDiagonalRatio=1 [Critical = 0.55; change with -CriticalDiagonalRatio]  
#  Forming stiffness matrixes took 0.014657 s 
FC: Viscosity structure info: min(0)=1.000000e+00 : max(687)=1.000000e+02 : contrast=1.000000e+02 
Using unpreconditioned (true) Stokes residual to define the FC stopping condition 
StokesResidual: Using L_2 
Stopping conditions: 
Velocity: RTOL: 1.000000e-06	ATOL: 1.000000e-16	minATOL: 1.000000e-05		fc_ksp_min_it 4
Pressure: RTOL: 1.000000e-06	ATOL: 1.000000e-16	minATOL: 1.000000e-05		fc_ksp_min_it 4
StokesResidual: Monitor activated 
# FC: using nonzero initial guess 
  Residual norms for fc_ solve.
  # KSP Iterations      /  L2_norm             scaled_L2_norm      Linf_norm           /  ... for each   2 blocks  [fc_] 
  0 KSP Residual norms  /  2.875456936497e+01  3.557273874628e-01  6.706875000000e-01  /  0.000000000000e+00  0.000000000000e+00  0.000000000000e+00  /  [fc_] 
  1 KSP Residual norms  /  5.412150226207e-01  6.695457810856e-03  1.595170532014e-02  /  3.906267868983e+00  1.220708709057e-01  2.924368635230e-01  /  [fc_] 
  2 KSP Residual norms  /  5.991090887036e-02  7.411674583776e-04  1.436124238770e-02  /  1.676437725685e-01  5.238867892765e-03  3.887867012070e-02  /  [fc_] 
  3 KSP Residual norms  /  6.343949967965e-02  7.848202209727e-04  1.062942003076e-02  /  9.790090505308e-02  3.059403282909e-03  2.265698217785e-02  /  [fc_] 
  4 KSP Residual norms  /  3.014703181770e-02  3.729537637010e-04  5.521079228519e-03  /  4.304028655445e-02  1.345008954826e-03  1.208035624990e-02  /  [fc_] 
  5 KSP Residual norms  /  1.853930808380e-02  2.293527524726e-04  3.831664745320e-03  /  3.038986486955e-02  9.496832771734e-04  8.040025636174e-03  /  [fc_] 
  6 KSP Residual norms  /  9.592339104385e-03  1.186683648762e-04  2.288387218324e-03  /  1.945663430749e-02  6.080198221090e-04  5.139809916567e-03  /  [fc_] 
  7 KSP Residual norms  /  6.123774950938e-03  7.575820166381e-05  1.001304373924e-03  /  9.425080109288e-03  2.945337534153e-04  3.209546311708e-03  /  [fc_] 
  8 KSP Residual norms  /  1.996354918681e-03  2.469722674881e-05  5.112279490407e-04  /  7.081512883039e-03  2.212972775950e-04  2.750136877906e-03  /  [fc_] 
  9 KSP Residual norms  /  2.096685701148e-03  2.593843494345e-05  5.502549770399e-04  /  4.015580961446e-03  1.254869050452e-04  1.485639942630e-03  /  [fc_] 
 10 KSP Residual norms  /  9.774584367884e-04  1.209229502480e-05  2.092129476017e-04  /  2.882048921474e-03  9.006402879606e-05  1.134275691341e-03  /  [fc_] 
 11 KSP Residual norms  /  7.679978174502e-04  9.501024122850e-06  1.214098761684e-04  /  1.826891428491e-03  5.709035714035e-05  7.201078652532e-04  /  [fc_] 
 12 KSP Residual norms  /  3.914857282496e-04  4.843132706027e-06  7.401796261798e-05  /  1.434713754441e-03  4.483480482627e-05  4.816548936316e-04  /  [fc_] 
 13 KSP Residual norms  /  4.173587603754e-04  5.163212134345e-06  9.049277358309e-05  /  1.007531849893e-03  3.148537030915e-05  3.311939753222e-04  /  [fc_] 
 14 KSP Residual norms  /  2.865688642320e-04  3.545189361299e-06  5.938258463963e-05  /  8.232696182151e-04  2.572717556922e-05  2.661375468105e-04  /  [fc_] 
 15 KSP Residual norms  /  2.225624869607e-04  2.753356206760e-06  3.572413268443e-05  /  6.268933641786e-04  1.959041763058e-05  1.939321571650e-04  /  [fc_] 
 16 KSP Residual norms  /  1.434659280856e-04  1.774839996387e-06  2.514676148483e-05  /  5.146095616441e-04  1.608154880138e-05  1.454110765414e-04  /  [fc_] 
 17 KSP Residual norms  /  1.080653072377e-04  1.336893240553e-06  1.971408608190e-05  /  4.351721983032e-04  1.359913119697e-05  1.117914194746e-04  /  [fc_] 
 18 KSP Residual norms  /  9.670011382997e-05  1.196292610871e-06  1.453732770872e-05  /  3.736049411283e-04  1.167515441026e-05  8.945571528275e-05  /  [fc_] 
 19 KSP Residual norms  /  8.206125426760e-05  1.015192932366e-06  1.729412348743e-05  /  3.221310413502e-04  1.006659504219e-05  7.760388246973e-05  /  [fc_] 
 20 KSP Residual norms  /  7.176769587798e-05  8.878496712944e-07  1.383048376763e-05  /  2.639965870392e-04  8.249893344976e-06  5.719687074502e-05  /  [fc_] 
 21 KSP Residual norms  /  6.599026084065e-05  8.163760962260e-07  1.259885904838e-05  /  2.299741527295e-04  7.186692272796e-06  4.787582196469e-05  /  [fc_] 
 22 KSP Residual norms  /  4.339938220391e-05  5.369007149070e-07  6.669038982199e-06  /  2.010676025174e-04  6.283362578667e-06  4.226072486908e-05  /  [fc_] 
 23 KSP Residual norms  /  2.770365517810e-05  3.427263595775e-07  4.607983060901e-06  /  1.798167023976e-04  5.619271949924e-06  2.962890225163e-05  /  [fc_] 
 24 KSP Residual norms  /  3.962597493055e-05  4.902192885867e-07  8.958438849515e-06  /  1.610170066088e-04  5.031781456526e-06  2.653450213724e-05  /  [fc_] 
 25 KSP Residual norms  /  3.042548967769e-05  3.763986105286e-07  6.813575415737e-06  /  1.443320308264e-04  4.510375963325e-06  2.625927722954e-05  /  [fc_] 
 26 KSP Residual norms  /  3.526153778542e-05  4.362261369704e-07  6.266559983229e-06  /  1.259259903205e-04  3.935187197514e-06  2.100133789316e-05  /  [fc_] 
 27 KSP Residual norms  /  2.586169349632e-05  3.199391563144e-07  6.489065486687e-06  /  1.094025970822e-04  3.418831158820e-06  1.794851089164e-05  /  [fc_] 
 28 KSP Residual norms  /  2.269341127241e-05  2.807438289926e-07  4.148078194798e-06  /  9.161870696548e-05  2.863084592671e-06  1.451948010072e-05  /  [fc_] 
 29 KSP Residual norms  /  1.708466610850e-05  2.113571433922e-07  3.424274967031e-06  /  8.229015756798e-05  2.571567423999e-06  1.154683455101e-05  /  [fc_] 
 30 KSP Residual norms  /  1.559218490115e-05  1.928934191058e-07  2.700447471953e-06  /  7.382030457707e-05  2.306884518033e-06  1.121702983150e-05  /  [fc_] 
 31 KSP Residual norms  /  9.468126868155e-06  1.171317153885e-07  1.620222267990e-06  /  7.038818113617e-05  2.199630660505e-06  9.523093954133e-06  /  [fc_] 
 32 KSP Residual norms  /  1.091944586486e-05  1.350862153679e-07  2.152623522420e-06  /  6.467925282010e-05  2.021226650628e-06  9.625363765195e-06  /  [fc_] 
 33 KSP Residual norms  /  1.189461901156e-05  1.471502387027e-07  2.400641726484e-06  /  5.755669791197e-05  1.798646809749e-06  1.012100201863e-05  /  [fc_] 
 34 KSP Residual norms  /  9.161499318695e-06  1.133383768165e-07  1.737003675765e-06  /  5.250075326332e-05  1.640648539479e-06  9.355729951394e-06  /  [fc_] 
 35 KSP Residual norms  /  1.071790913926e-05  1.325929722258e-07  1.987062894208e-06  /  4.886117769753e-05  1.526911803048e-06  8.330723777463e-06  /  [fc_] 
 36 KSP Residual norms  /  7.281382857393e-06  9.007915465890e-08  1.253916933804e-06  /  4.532900841230e-05  1.416531512884e-06  8.278331869786e-06  /  [fc_] 
 37 KSP Residual norms  /  9.921200409258e-06  1.227367608009e-07  2.024059729589e-06  /  4.150205686126e-05  1.296939276914e-06  7.414479830444e-06  /  [fc_] 
 38 KSP Residual norms  /  7.242880946836e-06  8.960284135088e-08  1.256625492549e-06  /  3.793691617886e-05  1.185528630589e-06  6.846831092603e-06  /  [fc_] 
 39 KSP Residual norms  /  8.192500742237e-06  1.013507400801e-07  1.341291051205e-06  /  3.405468633550e-05  1.064208947984e-06  6.492935671112e-06  /  [fc_] 
 40 KSP Residual norms  /  5.157298841079e-06  6.380177076611e-08  9.076883347324e-07  /  2.985501611746e-05  9.329692536705e-07  6.677730663808e-06  /  [fc_] 
 41 KSP Residual norms  /  6.378023987657e-06  7.890355725753e-08  1.313068847585e-06  /  2.623831375145e-05  8.199473047328e-07  5.227448052031e-06  /  [fc_] 
 42 KSP Residual norms  /  4.780753803814e-06  5.914347174351e-08  1.044758832560e-06  /  2.359257070241e-05  7.372678344502e-07  4.776106495628e-06  /  [fc_] 
 43 KSP Residual norms  /  4.439114738688e-06  5.491700009825e-08  9.082899133990e-07  /  2.114957617163e-05  6.609242553635e-07  4.139362971678e-06  /  [fc_] 
 44 KSP Residual norms  /  5.104759166796e-06  6.315179403255e-08  9.462779605893e-07  /  1.859585389858e-05  5.811204343305e-07  3.429673344684e-06  /  [fc_] 
 45 KSP Residual norms  /  2.680819389358e-06  3.316484644691e-08  4.950622955635e-07  /  1.645197366304e-05  5.141241769699e-07  3.024042479348e-06  /  [fc_] 
 46 KSP Residual norms  /  3.620157315175e-06  4.478554651910e-08  7.077801649999e-07  /  1.451055248466e-05  4.534547651456e-07  2.450661834641e-06  /  [fc_] 
 47 KSP Residual norms  /  2.634308208320e-06  3.258944916975e-08  4.669080966580e-07  /  1.325242798176e-05  4.141383744299e-07  1.870943359719e-06  /  [fc_] 
 48 KSP Residual norms  /  2.513127283933e-06  3.109030052678e-08  3.983729166263e-07  /  1.186546935009e-05  3.707959171904e-07  1.585553211176e-06  /  [fc_] 
 49 KSP Residual norms  /  2.069243844134e-06  2.559894733093e-08  3.478997388034e-07  /  1.060765259081e-05  3.314891434629e-07  1.442435287686e-06  /  [fc_] 
 50 KSP Residual norms  /  1.830800844330e-06  2.264913075387e-08  3.099128877006e-07  /  9.780061374348e-06  3.056269179484e-07  1.240383558530e-06  /  [fc_] 
 51 KSP Residual norms  /  1.092982983151e-06  1.352146770841e-08  2.186720051482e-07  /  8.951129577334e-06  2.797227992917e-07  8.388970328233e-07  /  [fc_] 
 52 KSP Residual norms  /  1.530805777435e-06  1.893784368697e-08  2.851111368313e-07  /  8.338515264307e-06  2.605786020096e-07  7.432995261993e-07  /  [fc_] 
 53 KSP Residual norms  /  1.288445562002e-06  1.593956660737e-08  2.985093069576e-07  /  7.682640013528e-06  2.400825004228e-07  8.286306623103e-07  /  [fc_] 
 54 KSP Residual norms  /  1.341483360697e-06  1.659570571789e-08  2.672238333897e-07  /  7.171663315730e-06  2.241144786166e-07  8.604917737954e-07  /  [fc_] 
 55 KSP Residual norms  /  1.442232343054e-06  1.784208753041e-08  3.166919117786e-07  /  6.515848149012e-06  2.036202546566e-07  8.026855021117e-07  /  [fc_] 
 56 KSP Residual norms  /  1.216168745501e-06  1.504541852322e-08  1.892899034908e-07  /  5.903005293264e-06  1.844689154145e-07  7.528534566517e-07  /  [fc_] 
 57 KSP Residual norms  /  1.413361478961e-06  1.748492144222e-08  2.515402684122e-07  /  5.074367596134e-06  1.585739873792e-07  6.046430309676e-07  /  [fc_] 
 58 KSP Residual norms  /  9.058654879627e-07  1.120660717730e-08  2.050729252507e-07  /  4.450102561917e-06  1.390657050599e-07  5.243189653558e-07  /  [fc_] 
 59 KSP Residual norms  /  8.767704896173e-07  1.084666828835e-08  1.733459790510e-07  /  4.091863864590e-06  1.278707457685e-07  4.234409970813e-07  /  [fc_] 
 60 KSP Residual norms  /  6.816830624603e-07  8.433210451139e-09  1.456003435591e-07  /  3.620072912348e-06  1.131272785109e-07  3.064840650497e-07  /  [fc_] 
 61 KSP Residual norms  /  5.978890486250e-07  7.396581272370e-09  1.105508116383e-07  /  3.494674350350e-06  1.092085734485e-07  2.911792573699e-07  /  [fc_] 
 62 KSP Residual norms  /  5.891346125598e-07  7.288278740322e-09  1.548785049771e-07  /  3.272198923916e-06  1.022562163724e-07  2.516959174159e-07  /  [fc_] 
 63 KSP Residual norms  /  6.658543462551e-07  8.237390865351e-09  1.568621999082e-07  /  3.091014575575e-06  9.659420548671e-08  2.458223179999e-07  /  [fc_] 
 64 KSP Residual norms  /  5.335229883834e-07  6.600298422142e-09  8.919255015303e-08  /  2.797764393904e-06  8.743013730951e-08  2.311189939186e-07  /  [fc_] 
Velocity convergence reason: RTOL 
Pressure convergence reason: RTOL 
Linear solve converged due to CONVERGED_HAPPY_BREAKDOWN iterations 64
#  Finished Stokes solve after 2.18736 s 
------------------------------------------
Stokes solver summary: 
  divergence: 
    Div_min = 1.110720173879e-10 
    Div_max = 2.311189941309e-07 
    |Div|_2 = 2.797764393716e-06 
    |Div|_1 = 6.627860566447e-05 
  momentum: 
    mRes_min = 0.000000000000e+00 
    mRes_max = 8.919363736482e-08 
    |mRes|_2 = 5.335232353198e-07 
    |mRes|_1 = 8.789462894105e-06 
  velocity: 
    u_min   = -7.424730346003e-01 
    u_max   = 4.618876067709e-01 
    |u|_2   = 5.779889075634e+00 
    |u|_1   = 2.383220162198e+02 
  pressure: 
    p_min   = -1.095828441709e+04 
    p_max   = 1.015387045340e+04 
    |p|_2   = 2.008035800769e+05 
    |p|_1   = 5.558690369875e+06 
------------------------------------------
The total memory used by integration points is 2.162688e+00 MB
The max memory used by integration points is 5.406720e-01 MB
The min memory used by integration points is 5.406720e-01 MB
#  Solving for velocity took 2.20586 s 
#  Forming temperature stiffness matrix mx,my,mz=(33, 2, 33) ...  finished. 
#  Forming temperature stiffness matrixes took 0.00430608 s 
#  Solving for temperature took 0.000627041 s 
The total memory used by integration points is 2.162688e+00 MB
The max memory used by integration points is 5.406720e-01 MB
The min memory used by integration points is 5.406720e-01 MB
The total memory used by particles is 7.741440e+00 MB
The max memory used by particles is 1.935360e+00 MB
The min memory used by particles is 1.935360e+00 MB
#  Updating properties @ integration points and particles took 0.025609 s 
AdvectionMethod = 0 
  Starting particle advection 
  Finished particle advection 
Minimum z-level phase 1 528309 
# Writing output to existing directory Timestep_000000 
# Saved MATLAB output in file Timestep_000000/Subduction3D_Output.0.1000000.out 
# Saved Paraview VTS Velocity output in file Subduction3D_Output_velocity_step000000-mesh.pvts 
# Saved Paraview VTS Temperature output in file Subduction3D_Output_temperature_step000000-mesh.pvts 
# Saved Paraview VTS Surface Topography output in file Subduction3D_Output_Surface_topography_step000000-mesh.pvts 
# Saved Paraview VTS Bottom Topography output in file Subduction3D_Output_Bottom_topography_step000000-mesh.pvts 
# Saved Paraview VTS point-wise quadrature output in file Subduction3D_Output_quadrature_step000000-mesh.pvts 
 Starting GetParticleNodes 
#rank=0, Attributing 6912 particles to elements took 0.00619888 s 
The total # of particles present is 27648 
 Finished GetParticleNodes 
 Maximum velocity  23.4307   [cm/year]  
 Time = 3168.81, dt=4673.36  [Years]  
# Finished timestep 0 out of 1 in 2.26393 s
  
# Total time required: 2.30248 s 
#PETSc Option Table entries:
-AddRandomNoiseParticles 0
-Air_ThicknessFactor 0.05
-GridAdvectionMethod 0
-MaximumSurfaceAngle 45
-ParamFile input/subduction3D.dat
-Slab_ThicknessFactor 0.1
-fc_ksp_converged_reason
-fc_ksp_max_it 1000
-fc_ksp_min_it 4
-fc_ksp_p_atol 1.0e-16
-fc_ksp_p_min_atol 1.0e-5
-fc_ksp_p_rtol 1.0e-6
-fc_ksp_type gcr
-fc_ksp_v_atol 1.0e-16
-fc_ksp_v_min_atol 1.0e-5
-fc_ksp_v_rtol 1.0e-6
-fc_pc_fieldsplit_schur_factorization_type UPPER
-fc_pc_fieldsplit_type SCHUR
-fc_pc_type fieldsplit
-incr_breakpoints 100
-options_left
-ps_ksp_type preonly
-ps_pc_type jacobi
-restart 0
-save_breakpoints 0
-time_end 1
-use_stokes_monitor
-use_stokes_norm L2
-use_stokes_residual
-vpt_element Q1P0
-vs_fieldsplit_1_pc_type sor
-vs_fieldsplit_ksp_type preonly
-vs_fieldsplit_mg_levels_pc_type sor
-vs_fieldsplit_pc_type ml
-vs_ksp_max_it 1000
-vs_ksp_rtol 1.0e-3
-vs_ksp_type gcr
-vs_pc_fieldsplit_block_size 3
-vs_pc_fieldsplit_type ADDITIVE
-vs_pc_type fieldsplit
#End of PETSc Option Table entries
There are no unused options.
